Fix NPE thrown when USING clause contained unknown columns (#15569)
crate/crate: docs/appendices/release-notes/5.6.3.rst

Fixes
=====

- Fixed ``NullPointerException`` thrown when joining tables with ``USING``

>   clause which contained columns not known to one side of the tables.

Reviewer: phrasing suggestion:
```suggestion
  clause which contains columns not existing in either or both tables.
```
At least `not known` -> `not existing` reads better imho.

Author: Thank you! updated.
